js/jsmind.screenshot.js

```diff
@@ -197,13 +227,23 @@
             ctx.fill();
 
             ctx.fillStyle = color;
-            if(text_overflow === 'ellipsis'){
-                jcanvas.text_ellipsis(ctx, node.topic, tb.x, tb.y, tb.w, tb.h);
-            }else{
-                var line_height = parseInt(css(ncs,'line-height'));
-                jcanvas.text_multiline(ctx, node.topic, tb.x, tb.y, tb.w, tb.h,line_height);
+            if (!!node.topic) {
+                if(text_overflow === 'ellipsis'){
+                    jcanvas.text_ellipsis(ctx, node.topic, tb.x, tb.y, tb.w, tb.h);
+                }else{
+                    var line_height = parseInt(css(ncs,'line-height'));
+                    jcanvas.text_multiline(ctx, node.topic, tb.x, tb.y, tb.w, tb.h,line_height);
+                }
+                node.ready = true;
+            }
+            if (!!node.image) {
+                var backgroundUrl = css(ncs,'background-image').slice(5, -2);
+                jcanvas.image(ctx, backgroundUrl, tb.x, tb.y, tb.w, tb.h,
+                    function() {
+                        node.ready = true;
+                    });
+                node.ready = false;
             }
```

Comment:
first jcanvas.text_ellipsis(...) then jcanvas.image(...), i think the text will be covered by the image, but without any verification, i will try later :)